src/components/MapboxMap/MapboxMap.vue

```diff
@@ -118,7 +142,18 @@ function initMap(accessToken: string) {
           promoteId: "spaceId",
         },
         layout: {
-          "icon-image": "marker-icon",
+          "icon-image": [
```

Comment:
I understand this is Mapbox syntax and you can't do anything about it, but should we add a comment like:
```js
// show marker for occupancy state and default to "map-marker"?
```